schema.js

```diff
@@ -3,20 +3,37 @@ const { buildSchema } = require('graphql');
 const schema = buildSchema(`
   type Query {
     routeSearch(from: Int, to: Int): [Route]
-    stationWith(evaId: Int): Station 
+    stationWith(evaId: Int): Station
     search(searchTerm: String): Searchable
     nearby(lat: Float, lon: Float): Nearby
     parkingSpace(id: Int): ParkingSpace
   }
-  
+
   type Searchable {
 	  stations: [Station]
+    operationLocations: [OperationLocation!]!
   }
-  
+
+  type OperationLocation {
```

Comment:
I thought there is a better translation but in fact it is the **official translation** of Deutsche Bahn. See also http://data.deutschebahn.com/dataset/sprachenmanagement